Updating the status of the data collection and the image of the host structures

bergazin opened the pull request.

davidlmobley commented on host_guest/cyclodextrin_derivatives/README.md:
```diff
@@ -6,7 +6,7 @@ For SAMPL7, the Gilson group is contributing binding data for two hosts (r-riman
 The challenge consists of predicting binding free energies of two guests to the series of modified cyclodextrin hosts shown below.
 
-![](images/sampl_host_structures.jpg)
+![](images/sampl_host_structures_coded.jpg)
```
I don't see that you added this file, just changed the link.

davidlmobley replied:
Also: If you DO add the file, would be good to replace the old file rather than adding a new file (which would just mean there are two different images of the same thing which can get confusing).

bergazin replied:
@davidlmobley Ok I added the the new images and deleted the old. Should be good now

bergazin commented:
@davidlmobley `sampl_host_structures_coded.jpg` has the name of each host listed underneath each structure and then `one sampl_host_structures_1.jpg` is just an image of the structures, no naming included. 

I just deleted the old host cdx file and added `sampl_host_structures_coded.cdx` (has names) and `sampl_host_structures_1.cdx` (just the host images, no names)

bergazin commented:
And I added two PDFs, `sampl_host_structures_coded.pdf` (has host names) and `sampl_host_structures_1.pdf` (no host names, just structures)

bergazin commented:
Structures only, no names
![sampl_host_structures_1](https://user-images.githubusercontent.com/29715177/57945811-b5f68800-788f-11e9-995f-9a9e63ab88e4.jpg)

Structures plus names
![sampl_host_structures_coded](https://user-images.githubusercontent.com/29715177/57945812-b5f68800-788f-11e9-8737-825912dc36a9.jpg)

davidlmobley commented:
OK, @bergazin I'll approve and merge this one. But ultimately we want to progress towards a layout where there is a README.md explaining what each file is and linking to it. Adding files without explaining what they are will make it harder for people who want to find what they want.

(Check out the SAMPL6 repo for something laid out well and very functional, e.g. see this page https://github.com/MobleyLab/SAMPL6/tree/master/host_guest which has a README.md (which gets automatically displayed when you visit it) which says what's in each file. As a result of the good organization on this repo, the better part of 100 different people have used the repo for things and we almost never field questions on it... So we want to progress towards that here.)
